lib/syskit/roby_app/ftp_server/server.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require "ftpd"
+require "ipaddr"
+
+require "syskit/roby_app/ftp_server/driver"
+
+module Syskit
+    module RobyApp
+        module FtpServer
+            # Class responsible for creating an FTP server
+            class Server
+                include Ftpd::InsecureCertificate
+
+                attr_reader :port
+
+                # tgt_dir must be an absolute path
+                def initialize(tgt_dir,
+                    auth_level: "password",
+                    user: ENV["LOGNAME"] || "test",
```

Comment:
Why is `ENV["LOGNAME"]` the default *user* ?

Also, don't add default arguments "only for testing" in the runtime code. Create yourself helpers in the tests to avoid repetitions (see later in the review).